draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1428,9 +1430,9 @@ Fixed-Length Dedicated Counter Length:
   the level of timestamp precision in use.
   For example, millisecond precision generally requires a larger counter than a
   timestamp with nanosecond precision.
-  General guidance is that the counter SHOULD be at least 12 bits but no longer
+  General guidance is that the counter should be at least 12 bits but no longer
```

Comment:
How about: "General guidance is that the counter SHOULD be at least 12 bits but no longer than 42 bits for UUIDv7."? This guidance isn't really effective without the v7 context, and with that context BCP 14 verbiage makes sense.